app/controllers/issues/merge_controller.rb

```diff
@@ -21,7 +21,9 @@ def create
       # create new issue if existing issue not given
       if @issue.new_record?
         @issue.author ||= current_user.email
-        if @issue.save && @issue.update_attributes(issue_params)
+        if @issue.save &&
+            (params[:issue] && @issue.update_attributes(issue_params))
```

Comment:
What was the reason for this change?

Reply:
For context, we build the form view params like: `{ item_form: [params] }` while we have the write view params as: `{issue: { text: ''} }`. I'd like the `item_form` to also be under the `{ issue: ... }` but this doesn't work in rails by default because it expects the `issue` instance to respond to `item_form` and to me, adding virtual attributes seems excessive. 

So, this means that when passing the form view for the issue merge, `params[:issue]` will not exist and `issue_params` will throw an error when invoked. We only try to call the `update_attributes` when the issue params exist. Otherwise, we used the "dradified" form.

Note that this does not happen in other controllers since we also pass other parameters even if the user uses the form view. For example, if the user submits the form view in the evidence, the evidence's issue is also passed so the overall params looks something like:
```
{ item_form: [data], evidence: { issue_id: ... } }
```